projects/storefrontlib/cms-components/misc/theme-switcher/theme-switcher.component.html

```diff
@@ -0,0 +1,18 @@
+<label *ngIf="items$ | async as themes">
+  <span> {{ 'themeSwitcher.theme' | cxTranslate }}:</span>
+  <select (change)="activeItem = $any($event).target.value">
+    <option
+      [attr.aria-label]="
+        theme.i18nNameKey ? (theme.i18nNameKey | cxTranslate) : theme.className
+      "
+      *ngFor="let theme of themes"
+      value="{{ theme.className }}"
+      [selected]="(activeItem$ | async) === theme.className"
+    >
+      {{
+        theme.i18nNameKey ? (theme.i18nNameKey | cxTranslate) : theme.className
+      }}
+    </option>
+  </select>
+  <cx-icon [type]="iconTypes.CARET_DOWN" class="small"></cx-icon>
```

Comment:
```suggestion
  <cx-icon [type]="iconTypes.CARET_DOWN" class="small"/>
```
See: https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-Useself-closingcomponenttags